db/migrate/20200429030034_enforce_uniqueness_at_database_level.rb

```diff
@@ -0,0 +1,8 @@
+class EnforceUniquenessAtDatabaseLevel < ActiveRecord::Migration[6.0]
+  def change
+    remove_index :games, :wikidata_id
+    add_index :games, :wikidata_id, unique: true
```

Comment:
Why remove the index, can't you just make it unique?

Reply:
Nope, gotta drop and then re-create.